docs/layers/identity/identity.mdx

```diff
@@ -25,110 +29,142 @@ demonstrating how it can be used to establish fine-grained access control for an
+
+    In addition, for the duration of an engagement, Cloud Posse configures our own IdP to access your infrastructure so that it’s easy for you to revoke our access when we’re done.
+  </Step>
+  <Step>
+    ### <StepNumber/> AWS IAM Roles with SAML is cumbersome
+
+    We needed to find a solution for machine access and to apply Terraform across accounts. To do that, we can use IAM roles with a SAML provider and assume them in Terraform or third party integrations such as GitHub Actions.
+
+    The challenge then becomes making it easy for users. AWS SAML provides low level controls and is a little more cumbersome to use, especially when you compare how easy it is for users to use the IAM Identity Center.
+
+    How can we have a consistent solution that works for both?
+  </Step>
+</Steps>
+<br/>
+
+AWS doesn’t make it easy to log in. Control Tower lacked APIs, until recently. Identity Center doesn’t work with automation. And IAM roles with SAML is cumbersome to use.
+
+It’s gotten a lot better with Identity Center, which is one of the reasons we recommend it, but it still doesn’t solve the problem of how to distribute configurations to developers to easily log in on their workstations or how to use IAM roles with SAML.
 
 ## Our Solution
 
-**Our goals: team oriented, highly accessible, easily configured**
+### Integrated with Single Sign On
+
+```mermaid
+flowchart LR
+    user["User"] --> idp["Identity Provider"]
+    idp --> identity_center["AWS IAM\nIdentity Center"]
+    identity_center --> aws_team["AWS Team"]
+    aws_team --> aws_team_role["AWS\nTeam Role"]
+```
+
+We use Identity Center to easily manage users and groups. These are connected with your Identity provider, such as Okta or GSuite.
 
-Taking into account all the challenges of using AWS SSO and SAML, let's summarize our requirements:
+Users can sign into Identity Center to access any account they have access to with fine grained access controls. Administrators have a single place to manage users and can use Terraform to define those fine grained access controls. Then we’ve bridged the gap between Identity Center and IAM roles
 
-- We need the ability to use AWS SAML or AWS IAM Identity Center
-- We need a single tool to manage logins to the AWS Console and CLI
-- We need the ability to switch roles into other accounts easily, with team and account specific permissions.
-- We need to be able to concurrently assume roles into multiple accounts (E.g. with Terraform)
+### Team Oriented
 
-### Team Oriented: Multiple AWS Accounts, One Team, with Exact Permissions
+```mermaid
+flowchart LR
+    subgraph identity_teams["identity Teams"]
```

Comment:
was the lower case "i" intentional?

Reply:
originally yes because it was in the account style - "core-identity". However I shortened that to the stage to look better in the mermaid graph, so I'll capitalize these too